tests/common.py

```diff
@@ -200,7 +200,7 @@ def assert_tifs_equal(tif1, tif2):
     d1 = mds.ReadAsArray()
     d2 = sds.ReadAsArray()
     # phase equal
-    np.testing.assert_array_almost_equal(d1,  d2, decimal=3)
+    np.testing.assert_array_almost_equal(d1,  d2, decimal=2)
```

Comment:
What have we changed that is making tests fail with 3 decimal places?

Reply:
Good question. This is un-resolved and I will document an Issue. The commit that introduced this drop in testing precision has been removed.